postman/techno-broadlink.postman_collection.json

```diff
@@ -152,4 +152,4 @@
 		}
 	],
 	"protocolProfileBehavior": {}
-}
\ No newline at end of file
+}
```

Comment:
It looks like you might have adjust spacing?  Not sure this commit needs to be here in this PR

Reply:
No worries!